src/package.test.ts

```diff
@@ -451,7 +453,7 @@ describe('package', () => {
       });
     });
 
-    it('updates the changelog of the package if requested to do so and if the package has one', async () => {
+    it('should migrate all unreleased changes to a release section.', async () => {
```

Comment:
Nit: We don't typically use "should" in tests (see our [guideline about naming tests](https://github.com/MetaMask/contributor-docs/blob/main/docs/unit-testing.md#use-it-to-specify-the-desired-behavior-for-the-code-under-test)):

``` suggestion
    it('migrates all unreleased changes to a release section', async () => {